apps/tangle-dapp/containers/BondMoreTxContainer/BondTokens.tsx

```diff
@@ -39,7 +39,7 @@ const BondTokens: FC<BondTokensProps> = ({
       </InputField.Root>
 
       <Typography variant="body1" fw="normal">
-        Enter the amount you would like to add to your current stake.
+        Added stake will be bonded and subject to a 28-Era (apprx. 28 days) unbonding period before withdrawal is possible.
```

Comment:
The approximate timing depends on chain parameters (ie. session length, sessions per era, etc.) so might be best to remove the estimate to avoid confusing users if those parameters change in the future, which would render the approximation invalid

Reply:
Got it, I suppose currently we don't have a way to dynamically query these parameters?